Cloud Pak for Data/WML/notebooks/binary/spark/Watson OpenScale and Watson ML Engine.ipynb

```diff
@@ -24,7 +24,7 @@
     "id": "ab7073b7-baf8-48ba-b9fb-e39e497596aa"
    },
    "source": [
-    "This notebook should be run using with **Default Spark 3.0 & Python 3.8** runtime environment. **If you are viewing this in Watson Studio and do not see Python 3.8.x in the upper right corner of your screen, please update the runtime now.** It requires service credentials for the following services:\n",
+    "This notebook should be run using with **Default Spark 3.0 & Python 3.9** runtime environment. **If you are viewing this in Watson Studio and do not see Python 3.9.x in the upper right corner of your screen, please update the runtime now.** It requires service credentials for the following services:\n",
```

Comment:
The Spark version should be changed to 3.2 because that's what is given below `spark-mllib_3.2`